### DISCUSSION TITLE
GitHub student developer pack Expired and i am still a student

### DISCUSSION THREAD

**Discussion by Yasshu55** [community/community#103765] [GitHub Education]
### Hi everyone,

I am a 2024 grad who will graduate in June so recently my GitHub student pack expired and I tried to reavail it but it was rejected. So how can I get access to the student developer pack as I am still a student for 3-4 months?

**Comment by Cauatn** [comment]
I find a similar discussion in the community, hope that can help you.
https://github.com/orgs/community/discussions/30170

**Reply by Yasshu55** [reply]
No it doesnt :(

**Comment by mishmanners** [comment]
What is the error message or response you are receiving when you resubmit?

**Reply by Yasshu55** [reply]
It's telling me to fill in the billing info...this is the mail that i received after i reapplied (i tried twice with different pics of my college id )
![Screenshot (384)](https://github.com/community/community/assets/95294297/1e864ab4-0c6a-44d3-a094-9f536ee4e5b4)

**Reply by mishmanners** [reply]
Firstly, ensure you secure your account with 2FA. Try applying again by taking the photo of your academic documentation with the device camera rather than uploading it.

**Comment by Yasshu55** [comment]
Hello, can anyone help me with whom to contact to resolve this issue? like any support ticket i can raise or something like that

**Reply by mishmanners** [reply]
You can always raise a support ticket, however it's declining your academic documentation, so I'd firstly reapply and take the photo with the device rather than uploading the document.
